package.json

```diff
@@ -6,6 +6,8 @@
   "dependencies": {
     "body-parser": "^1.20.0",
     "node-fetch": "^2.6.7",
+    "express-http-context": "~1.2.4",
+    "sparql-client-2": "https://github.com/erikap/node-sparql-client.git",
```

Comment:
I see you added this dependency that is no longer maintained
https://www.npmjs.com/package/sparql-client-2